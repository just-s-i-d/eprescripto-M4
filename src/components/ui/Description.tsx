type CardTitlePropsType = {
  children: React.ReactNode;
  className?: string;
};
const Description = ({ children, className }: CardTitlePropsType) => {
  return (
    <h1 className={`text-sm font-bold mb-1 text-slate-400 ${className}`}>
      {children}
    </h1>
  );
};
export default Description;
