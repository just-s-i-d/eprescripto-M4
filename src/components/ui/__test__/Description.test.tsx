import { render, screen } from "@testing-library/react";
import "@testing-library/jest-dom";
import Description from "../Description";

describe("Description Test", () => {
  const description = "Test Description";
  const className = "test";
  it("Title test", () => {
    render(<Description className={className}>{description}</Description>);
    const descriptionElement = screen.getByText(description);
    expect(descriptionElement).toBeInTheDocument();
    expect(descriptionElement).toHaveClass(className);
  });
});
