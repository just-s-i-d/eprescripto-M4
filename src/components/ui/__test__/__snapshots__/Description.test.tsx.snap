// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Description Test Snapsot test 1`] = `
<DocumentFragment>
  <h1
    class="text-sm font-bold mb-1 text-slate-400 test"
  >
    Test Description
  </h1>
</DocumentFragment>
`;
